samples/DataSourceWithProcessingAndExtensions/Sample/MetadataTables/EventStatsTable.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Performance.SDK.Extensibility;
+using Microsoft.Performance.SDK.Processing;
+using DataExtensionsSample.SourceDataCookers;
+
+namespace DataExtensionsSample.MetadataTables
+{
+    [Table]
+    [RequiresCooker(EventStatsCooker.CookerPathAsString)]
+    public class EventStatsTable
+    {
+        public static TableDescriptor TableDescriptor = new TableDescriptor(
+            Guid.Parse("E6B3FE19-7E1F-456C-AFC8-25A998B01F53"),
+            "Event Stats",
+            "Event Stats",
```

Comment:
Use a human-readable description now that WPA uses names correctly. Same for other tables